src/BinDeps.jl

```diff
@@ -523,8 +524,14 @@ function run(s::SynchronousStepCollection)
     end
 end
 
-is_unix() && (make_command = `make -j8`)
-is_windows() && (make_command = `make`)
+const make_command =
```

Comment:
Yes, I saw all of that. I've exported `make_command` as part of this PR so that build steps can use
```
`$make_command`
```
rather than
```
`make`
```